cmd/pairing/pairing.go

```diff
@@ -84,29 +83,5 @@ func pairingPersistentPreRunE(cmd *cobra.Command, args []string) error {
 }
 
 func generatePairingJWT(privateKey string) (jwtString string, err error) {
-	keyPEM, err := ioutil.ReadFile(privateKey)
-	if err != nil {
-		return "", err
-	}
-
-	key, err := jwt.ParseRSAPrivateKeyFromPEM(keyPEM)
-	if err != nil {
-		return "", err
-	}
-
-	now := time.Now().UTC().Unix()
-	// 5 minutes expiry
-	expiry := now + 300
-	token := jwt.NewWithClaims(jwt.SigningMethodRS256, jwt.MapClaims{
-		"a_pa": []string{"^.*$::^.*$"},
-		"iat":  now,
-		"exp":  expiry,
-	})
-
-	tokenString, err := token.SignedString(key)
-	if err != nil {
-		return "", err
-	}
-
-	return tokenString, nil
+	return utils.GenerateAstarteJWTFromKeyFile(privateKey, utils.Pairing, nil, 30)
```

Comment:
I would use 5 minutes (300 seconds) for token ttl to avoid problems with clock skew between client and server